docs/architecture/commitlog/fileformat.rst

```diff
@@ -0,0 +1,60 @@
+ScyllaDB commitlog segment file format
+======================================
+
+File descriptor structure
+.........................
+
```

Comment:
Please start with a note that commitlog formats are subject to change from version to version and users should not rely on them.

Reply:
Done.